except (ValueError, KeyError):
    print("A introducido un valor err√≥neo o inexistente.")
    